samples/bluetooth/fast_pair/input_device/boards/nrf54h20dk_nrf54h20_cpuapp.overlay

```diff
@@ -4,9 +4,9 @@
  * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
  */
 
-&mram1x {
-	/delete-node/ cpuapp_rw_partitions;
```

Comment:
btw. `/delete-node/ cpuapp-rw-partitions;` syntax seems to work here (just a curious note, your approach is fine too - no need to change)

Reply:
I get this error: `devicetree error: Label 'storage_partition' appears on /soc/mram@e000000/cpuapp-rw-partitions/partition@1e3000 and on /soc/mram@e000000/cpuapp-rw-partitions/partition@1e4000`

Reply:
It seems that the earlier definition is not deleted.

Reply:
Saw it for `/delete-node/ cpuapp_rw_partitions;`. Changing to `/delete-node/ cpuapp-rw-partitions;` did the trick for me.